test/lib/mockerode.ts

```diff
@@ -641,25 +722,114 @@ export class MockEngine {
 		this.volumes[Name] = volume;
 	}
 
-	getImage(id: string) {
-		const image = Object.values(this.images).find(
-			(img) =>
-				img.inspectInfo.Id === id ||
-				img.inspectInfo.RepoTags.some(
-					(tag) => tag === id || tag === id + ':latest',
-				) ||
-				img.inspectInfo.RepoDigests.includes(id),
+	// NOT a dockerode method
+	// Implements this https://docs.docker.com/engine/reference/commandline/rmi/
+	// Removes (and un-tags) one or more images from the host node.
+	// If an image has multiple tags, using this command with the tag as
+	// a parameter only removes the tag. If the tag is the only one for the image,
+	// both the image and the tag are removed.
+	// Does not remove images from a registry.
+	// You cannot remove an image of a running container unless you use the
+	// -f option. To see all images on a host use the docker image ls command.
+	//
+	// You can remove an image using its short or long ID, its tag, or its digest.
+	// If an image has one or more tags referencing it, you must remove all of them
+	// before the image is removed. Digest references are removed automatically when
+	// an image is removed by tag.
+	removeImage(name: string, { force } = { force: false }) {
```

Comment:
Is this a convenience method for for Dockerode's `docker.getImage(img.dockerImageId).remove({ force: true })` to make tests simpler?

Reply:
Never mind, I understand why now.